### ISSUE TITLE
custom loss function load error

### ISSUE THREAD

**Issue by jsmith173** [keras-team/keras#19031]
I've a model with custom loss function. Is it possible to save the model after 'compile and train' and load without compiling and training again? I can't compile and train the model after each loading... I'd like to run inference only after loading. So maybe I need the weights only? Is it possible to load the model to run only a inference?

**Comment by jsmith173** [comment]
I've a custom loss function,

**Comment by sachinprasadhs** [comment]
For the custom loss, you can follow one of the methods to map the custom objects for loading the model and then you can predict normally.

```
1. Registering custom objects (preferred),
2. Passing custom objects directly when loading, or
3. Using a custom object scope
```

For details of usage on the above topic, you can refer the document [here](https://keras.io/guides/serialization_and_saving/#custom-objects).
